src/js/index.js

```diff
@@ -35,7 +35,7 @@ store.subscribe(throttle(model.update, 16));
 var ui = require('./components');
 
 // name model Scene
-model.Scene.name = 'Scene';
+model.Scene._name = 'Scene';
```

Comment:
if we have a .name() to get the name we should have a setName() too,  also change name() to getName() ?